src/services/EcoBridge/Connext/Connext.reducer.ts

```diff
@@ -0,0 +1,104 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { TokenList } from '@uniswap/token-lists'
+
+import { BridgeDetails, BridgingDetailsErrorMessage, ConnextList, SyncState } from '../EcoBridge.types'
+import { connextTransactionsAdapter } from './Connext.adapter'
+import { ConnextBridgeState, ConnextTransaction, ConnextTransactionStatus } from './Connext.types'
+
+const initialState: ConnextBridgeState = {
+  transactions: connextTransactionsAdapter.getInitialState({}),
+  bridgingDetails: {},
+  bridgingDetailsStatus: SyncState.IDLE,
+  listsStatus: SyncState.IDLE,
+  lists: {},
+  lastMetadataCt: 0,
```

Comment:
```suggestion
  lastMetadataCounter: 0,
```

Reply:
In future it will be unified and moved to common reducer. Same for `setBridgeDetails`

Reply:
Perfect. Looking forward to it :)